float64 c22
float64 c21
float64 c20
float64 c12
float64 c11
float64 c10
float64 c02
float64 c01
float64 c00
bool use_abs
---
